packages/extension/src/newtab/DndModal.tsx

```diff
@@ -105,23 +114,22 @@ const DoNotDisturbModal: React.FC<DoNotDisturbModalProps> = ({
       }}
     >
       <ModalCloseButton onClick={onRequestClose} />
-      <div className={styles.heading}>
-        <h3 className={styles.title}>Do Not Disturb</h3>
-        {isActive ? null : (
-          <p className={styles.description}>
-            Choose your preferences while you&apos;re on Do Not Disturb mode
-          </p>
-        )}
-      </div>
-      {renderForm()}
-      <div className={classnames(styles.footer, styles.centered)}>
-        <button
-          className={classnames(styles.done, styles.centered)}
-          onClick={(e) => (isActive ? setDndSettings(null) : handleSubmit(e))}
-        >
-          {isActive ? 'Turn off' : 'Done'}
-        </button>
+      <div className="px-16 text-center">
+        <h3 className="font-bold text-2xl text-white">Do Not Disturb</h3>
```

Comment:
We usually don't use the tailwind prebuilt classes for typography and classes. We have our own.
`font-bold` is great in this case.
But `text-2xl` should be replaced with `typo-title3`. You can see all the available typography classes here:
https://github.com/dailydotdev/apps/blob/master/packages/shared/tailwind/typography.js
`text-white` should be replaced with `text-theme-label-primary` which sets the text color to `var(--theme-label-primary). This takes into account the current theme of the user.

Reply:
Thank you so much for the reference! It amazes me so much how well-built this app is.